tests/TenantDatabaseManagerTest.php

```diff
@@ -222,6 +223,159 @@
 
     $manager->setConnection('mysql2');
     expect($manager->databaseExists($name))->toBeTrue();
+
+    $manager->deleteDatabase($tenant);
+    expect($manager->databaseExists($name))->toBeFalse();
+});
+
+test('tenant database user can be created when using the permission controller MySQL manager', function () {
```

Comment:
Doesn't the previous test already do this?

Also, the previous test should likely be fixed. It said `tenant database can be created on a foreign server` **but this behavior was broken so the test doesn't actually verify that**. Therefore the code of the test should likely be rewritten.

Reply:
This review and https://github.com/archtechx/tenancy/pull/946#discussion_r982287654 are kinda connected. I explained there why I added this test.

Reply:
> Doesn't the previous test already do this?

I see the previous test (test with dataset) is doing the same. So I'm removing it.

Reply:
Can this be resolved now?

Reply:
Yes.